src/components/ToDoItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './ToDoItem.css'
+
+function ToDoItem() {
+	return (
+		<div className="todo-item">
+			<input type="checkbox" />
+        	<p>Task text will be here</p>
```

Comment:
please keep valid tabulation

Reply:
I'm trying to fix it, but on local computer it's all good, when i want to change and save it, git doesn't see the change

Reply:
You never have to fix it on github. 
1. fix locally
2. commmit locally
3. push to this branch
4. notify me
Thanks

Reply:
Fixed it.